src/listener.js

```diff
@@ -17,7 +17,7 @@ const {
  * Attempts to close the given maConn. If a failure occurs, it will be logged.
  *
  * @private
- * @param {import('libp2p-interfaces/src/transport/types').MultiaddrConnection} maConn
+ * @param {MultiaddrConnection} maConn
```

Comment:
We need to keep the previous type. Otherwise, the TS compile does not know what MultiaddrConnection is

Reply:
Its been defined on top of the file now.